examples/Nami.purs

```diff
@@ -13,10 +13,11 @@ import Contract.Address (getWalletAddress, getWalletCollateral)
 import Contract.Monad
   ( Contract
   , defaultContractConfig
-  , launchAff_
```

Comment:
Why remove this? It's re-exported from `Contract.Monad` so users don't have to add `aff` to their dependencies in spago.dhall